storage/remote/otlptranslator/prometheusremotewrite/metrics_to_prw_test.go

```diff
@@ -28,9 +28,9 @@ import (
 	"go.opentelemetry.io/collector/pdata/pmetric"
 	"go.opentelemetry.io/collector/pdata/pmetric/pmetricotlp"
 
+	prometheustranslator "github.com/prometheus/otlp-prometheus-translator"
```

Comment:
The linter should have caught it, but this import should be moved to the block starting at line 25, i.e. non-stdlib/non-local imports.